tests/Math.py

```diff
@@ -33,7 +33,26 @@ def testWhere(self):
         for i in range(10):
             if i != 5:
                 self.assertEqual(int1DArrRes[i], 0)
-            
+                
+    def testPolyFit(self):
+        xArr = np.ndarray(shape=(5), dtype=np.float64)
+        yArr = np.ndarray(shape=(5), dtype=np.float64)
+        for i in range(5):
+            xArr[i] = i
+            yArr[i] = (i*3.)+1.
+        yArr[3] = -10.
+        print 'yArr = ',yArr
+        print 'type(xArr[0]) = ',type(xArr[0])
+        nDeg = 2#c_size_t(3)
+        print 'type(nDeg) = ',type(nDeg)
+        coeffs = drpStella.PolyFit(xArr, yArr, nDeg)
+        print 'coeffs = ',coeffs
+        
+        coeffs = drpStella.PolyFit(xArr, yArr, nDeg, -3., 3., 2)
+        print 'coeffs = ',coeffs
+
+        self.assertTrue(drpStella.testPolyFit())
```

Comment:
I'm confused by what's going on here.

It seems like you first do some calculations in Python, but never check their results. Then you delegate the whole of the test to C++ code. Is that correct? If so, please don't.

First of all, drop the printing unless it's useful.

Next, please check the coeffs returned in Python for validity.

At least some of the tests in drpStella.testPolyFit seem to be degenerate with the Python code (for example, the Python call at line 48 above seems to be basically equivalent to the C++ at line 425 of Utils.cc). Please drop all of these redundant tests.

If it's possible to test all of the PolyFit functionality in pure Python code, please do that. If it's not possible, you can add a minimal C++ test which just exercises the functionality which isn't exposed to Python, and add comments explaining 1) that that's what you're doing, 2) why the relevant functionality isn't available in Python.